scanner/src/test/kotlin/storages/ClearlyDefinedStorageTest.kt

```diff
@@ -359,7 +363,7 @@ class ClearlyDefinedStorageTest : WordSpec({
 
             val storage = ClearlyDefinedStorage(storageConfiguration(wiremock))
 
-            assertEmptyResult(storage.read(id), expId = id)
+            assertFailure(storage.read(id))
```

Comment:
FYI, *kotest* has `BeSuccess()` / `BeFailure()` matchers (note the upper-case "B") for Kotlin `Result`s.

Reply:
Yes, but these are not Kotlin's classes but our own.

Reply:
Ah, right. I always get confused that we have our own `Result` class...